packages/gatsby-theme-docs/src/layouts/internals/layout-main.js

```diff
@@ -26,7 +26,13 @@ const Container = styled.main`
 `;
 
 const LayoutMain = (props) => (
-  <Container preventScroll={props.preventScroll}>
+  <Container
+    preventScroll={props.preventScroll}
+    // This is important to make the main container the active element when the page loads.
+    // The user can then use the keyboard navigation for example to scroll up/down.
+    autoFocus
```

Comment:
Works in chrome but not firefox.  It seems one still has to call `node.focus()` in javascript to focus a specific node in firefox.